src/features/authentication/forgot_password/forgotPassword.scss

```diff
@@ -0,0 +1,126 @@
+.forgot__password {
```

Comment:
The outer div represents the Block in the BEM convention, so a better class naming would be `forgot-password`.
All the other classes can be chained with `&`. E.g. `&__card` instead of `.forgot__card`, which translates to `forgot-password__card`.

Reply:
I have a problem when doing that: my styles do not apply anymore. Color, border-radius, background do not apply on the elements. They do apply only if I set them to "!important".